### PR TITLE
TMA-639: Check downloaded file size

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/566/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/356/ : SUCCESS in 1m 48s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/661/ : SUCCESS in 1m 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/567/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/357/ : SUCCESS in 1m 43s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/662/ : SUCCESS in 1m 41s

File: lib/gooddata/helpers/data_helper.rb [hunk 1]
```diff
@@ -114,6 +114,13 @@ def realize_s3(params)
         bucket = s3_client.bucket(bucket_name)
         obj = bucket.object(key)
         obj.get(response_target: filename, bucket: bucket_name, key: key)
+        s3_size = obj.size
+        actual_size = File.size(filename)
+        puts "File size in S3: #{s3_size}"
```
Comment (kubamahnert) [inline lib/gooddata/helpers/data_helper.rb]: why use puts instead of a logger?

File: lib/gooddata/helpers/data_helper.rb [hunk 2]
```diff
@@ -109,12 +109,19 @@ def realize_s3(params)
         key = @options[:key]
         raise 'Key "bucket" is missing in S3 datasource' if bucket_name.blank?
         raise 'Key "key" is missing in S3 datasource' if key.blank?
-        puts "Realizing download from S3. Bucket #{bucket_name}, object with key #{key}."
+        GoodData.logger.info("Realizing download from S3. Bucket #{bucket_name}, object with key #{key}.")
```
Comment (yenkins) [inline lib/gooddata/helpers/data_helper.rb]: Calls 'gooddata.logger' 4 times (DuplicateMethodCall)

**Comment by kubamahnert** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/574/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/364/ : SUCCESS in 34s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/669/ : SUCCESS in 1m 51s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/231/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/365/ : SUCCESS in 1m 44s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/670/ : SUCCESS in 1m 54s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1130/ : SUCCESS in 47m 53s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/887/ : SUCCESS in 28m 58s

**Pull request merged**